plugin/pkg/scheduler/algorithm/priorities/taint_toleration.go

```diff
@@ -57,6 +57,12 @@ func ComputeTaintTolerationPriorityMap(pod *v1.Pod, meta interface{}, nodeInfo *
 	if node == nil {
 		return schedulerapi.HostPriority{}, fmt.Errorf("node not found")
 	}
+	// get taints from node info cache.
+	taints, err := nodeInfo.Taints()
+	if err != nil {
+		// fallback to computing taints from node Spec
+		taints = node.Spec.Taints
```

Comment:
In case we cannot get values from node info cache, we are computing it from node spec again.